pdr_backend/lake/table_pdr_payouts.py

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright 2024 Ocean Protocol Foundation
+# SPDX-License-Identifier: Apache-2.0
+#
+from polars import Int64, Float64, Utf8, Boolean
+
+payouts_table_name = "pdr_payouts"
+
+# RAW PAYOUT SCHEMA
+payouts_schema = {
+    "ID": Utf8,
+    "token": Utf8,
+    "user": Utf8,
+    "slot": Int64,
+    "timestamp": Int64,
+    "payout": Float64,
+    "predictedValue": Boolean,
+    "revenue": Float64,
+    "roundSumStakesUp": Float64,
+    "roundSumStakes": Float64,
```

Comment:
The roundSumStakes values are not shown inside the diagram, should they be removed? Also saw them in the subgraph payout query

Reply:
```
The roundSumStakes values are not shown inside the diagram, should they be removed? Also saw them in the subgraph payout query
```
I have not changed the fetch or the schema because I did not want to break anything that's currently dependent on it

"The Diagram" is an up-to-date reflection of what the ETL generates.

Reply:
Isn't the payouts table part of the Lake/ETL?